VogtObserver/Sum.cs

```diff
@@ -4,7 +4,20 @@
 
 namespace VogtObserver
 {
-    class Sum
+    class Sum : IObserver
```

Comment:
`Sum` as a class name is throwing me off. I think because this wouldnt be a real world example -- usually you don't have a Sum object observing a random number generator. Your other example (Observer2) is a lot more real world. Traders observing a stock ticker so that they're notified when it changed.

Trying to think of some better names but none are coming to mind yet. The name `Sum` still seems off though.